spec/requests/rich_text_reactions_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'RichTextReaction requests' do
+  let!(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  describe 'POST /rich_text_reactions' do
+    it 'creates a rich text reaction' do
+      expect {
```

Comment:
For readability I would write the two `expect` lines at the end of the `it 'creates a rich text reaction' do` block.

Reply:
I think I know what you’re suggesting but I want to be clear.
Are you suggesting I take out the code for `post` and run that first, and then `expect` the count to change?
Something like this?
```
count = RichTextReaction.count
post rich_text_reaction_url, …
expect(RichTextReaction.count).to eq(count + 1)
```

Reply:
Exactly :)